backend/sozisel/test/sozisel_web/schema/participants/participant_mutations_test.exs

```diff
@@ -427,5 +442,88 @@ defmodule SoziselWeb.Schema.ParticipantMutationsTest do
                }
              } = run_query(ctx.conn, @submit_poll_result_mutation, variables)
     end
+
+    test "submit whiteboard result", ctx do
+      template = insert(:template)
+      event = insert(:whiteboard_event, session_template_id: template.id)
+      session = insert(:session, session_template_id: template.id)
+      launched_event = insert(:launched_event, event_id: event.id, session_id: session.id)
+      participant = insert(:participant, session_id: session.id)
+
+      File.copy!("test/assets/test_image.png", "/tmp/test_image.png")
+
+      upload = %Plug.Upload{
+        content_type: "image/png",
+        path: "/tmp/test_image.png",
+        filename: "whiteboard_image.png"
+      }
+
+      variables = %{
+        input: %{
+          launched_event_id: launched_event.id,
+          image: "image",
+          text: "some text",
+          used_time: 145
+        },
+        token: participant.token
+      }
+
+      assert %{
+               "data" => %{
+                 "submitWhiteboardResult" => %{
+                   "id" => _,
+                   "resultData" => %{
+                     "used_time" => 145.0,
+                     "text" => "some text"
+                   }
+                 }
+               }
+             } =
+               ctx.conn
+               |> post("/api/",
+                 query: @submit_whiteboard_result_mutation,
+                 variables: variables,
+                 image: upload
+               )
+               |> json_response(200)
+
+      assert Disk.file_exists?("whiteboard_#{launched_event.id}_#{participant.id}.png")
```

Comment:
can you use the function that generates the file name in some module I don't remember? Because you can mistype here something by hand and that function should be the only source of filename I guess.

Reply:
done